src/UnityPackage/Assets/Editor/ServerSettings/ChainsafeServerSettings.cs

```diff
@@ -125,4 +137,30 @@ void OnGUI()
         }
         GUILayout.Label("Reminder: Your ProjectID Must Be Valid To Save & Build With Our SDK. You Can Register For One On Our Website At Dashboard.Gaming.Chainsafe.io", EditorStyles.label);
     }
+
+    public void WriteNetworkFile()
+    {
+        // declares paths to write our javascript files to
+        string path1 = "Assets/WebGLTemplates/Web3GL-2020x/network.js";
```

Comment:
The templates shouldn't be hard-coded, the code should list the folders inside `WebGLTemplates` and create the file in all folders that are actually there.

Reply:
how do you mean?

Reply:
or this 

```
string root = "Assets/WebGLTemplates/";
var info = new DirectoryInfo(root);
var fileInfo = info.GetFiles();
```

Reply:
also if you noticed, both network files are different, so i'm not sure if a for x for each file work

Reply:
@sneakzttv @Arshia001 might mean with regards to `Web3GL-2020x`, there might be additional or different version templates in there, meaning this should be potentially created in all templates that appear to require the file, not just the one expected.

If I'm reading that correctly